frontend/src/pages/modelServing/screens/metrics/DIRChart.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Stack, StackItem } from '@patternfly/react-core';
+import { InferenceMetricType } from '~/pages/modelServing/screens/metrics/ModelServingMetricsContext';
+import TrustyChart from '~/pages/modelServing/screens/metrics/TrustyChart';
+import { DomainCalculator, MetricsChartTypes } from '~/pages/modelServing/screens/metrics/types';
+
+const DIRChart = () => {
+  const DEFAULT_MAX_THRESHOLD = 1.2;
+  const DEFAULT_MIN_THRESHOLD = 0.8;
+
+  const domainCalc: DomainCalculator = (maxYValue) => ({
+    y: maxYValue > 1.2 ? [0, maxYValue + 0.1] : [0, 1.3],
```

Comment:
More use of Threshold stuffs instead of direct values?

Reply:
Again, having threshold as part of the calculation doesn't make sense to me -- the threshold has no bearing on what your max or min value will be and the graph's domain should be relative to that.

Reply:
Following your comment below and fully grokking the formulae, I've reworked this one too.